### PR TITLE
Add `Skinning Contest: Unchained` news post

### PR DESCRIPTION
**Pull request by 0x84f**
- [x] Mention the new "fixed" schedule in the news post
- [x] make the contest page and check ID in news post
- [x] @RockRoller01 seal of approval
- [x] re-time
- [x] move threads to visible forum right before the post goes live

[web preview](https://osu3.roanh.dev/home/news/2024-04-09-skinning-contest-unchained)

### CODE REVIEW

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Voting
+
+After the submission phase is completed, the voting phase will be announced via a separate news post with more details. The voting will stay open for 14 days, and once the voting closes the full voting results will be visible on the contest page, and the top 3 participants will receive their prizes.
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Announcement | 2024-04-07 |
+| Submission phase | 2024-04-07/2024-05-12 |
```
Comment (RockRoller01) [inline news/2024/2024-04-09-skinning-contest-unchained.md:37]:
```suggestion
| Submission phase | 2024-04-09/2024-05-14 |
```
Start is getting pushed back, so the schedule also needs to

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+date: 2024-04-09 15:00:00 +0000
+---
+
+Unshackle your creativity and showcase it to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team have thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) the skinning community will once again be able to choose from a huge variety of topics and themes, whether it'd be anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
```
Comment (RockRoller01) [inline news/2024/2024-04-09-skinning-contest-unchained.md:13]: > we also should expand that its not just the theme that is relaxed, but also the permission rules

(^quote from skinship staff chat)
I currently see no mention of this in this section at all

**Review by RockRoller01 — CHANGES_REQUESTED** [review]
only gave it a quick glance, will do a full review once I am back from work

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Unshackle your creativity and showcase it to its fullest by participating in the newest skinning contest!
```
Comment (peppy) [inline news/2024/2024-04-09-skinning-contest-unchained.md:7]: Reads pretty weirdly, on first read my question is "showcase what?". Maybe just me.

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+The skinship team have thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) the skinning community will once again be able to choose from a huge variety of topics and themes, whether it'd be anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
+
+Regardless of whether you are a seasoned skinning contest veteran or if this is your first time joining us, let's first get familiar with the proper way to submit your beautifully crafted masterpieces, and the rules of the contest!
+
+## Participation
+
+As per the usual, all skins have to be created from scratch and be aligned with the rules listed below!
```
Comment (peppy) [inline news/2024/2024-04-09-skinning-contest-unchained.md:19]: Including the anime girl illustrations right? I'd hope that's the case.

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-04-09-skinning-contest-unchained.md (hunk 4 above)
Comment (RockRoller01) [reply news/2024/2024-04-09-skinning-contest-unchained.md:19]: Its an asset, so yes, they need to be credited

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Showcase your unshackled creativity to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
```
Comment (ubinici) [inline news/2024/2024-04-09-skinning-contest-unchained.md:11]: I would seek out ways to remove the latter line and add an emphasis that there are no theme restrictions for this iteration to the thumbnail text.

Comment (0x84f) [reply news/2024/2024-04-09-skinning-contest-unchained.md:11]: 76dd24902877a115a3ca529ed69c46a0d4cc8f65 should be addressed here

File: news/2024/2024-04-09-skinning-contest-unchained.md (hunk 4 above)
Comment (0x84f) [reply news/2024/2024-04-09-skinning-contest-unchained.md:23]: 76dd24902877a115a3ca529ed69c46a0d4cc8f65

File: news/2024/2024-04-09-skinning-contest-unchained.md (hunk 2 above)
Comment (0x84f) [reply news/2024/2024-04-09-skinning-contest-unchained.md]: 76dd24902877a115a3ca529ed69c46a0d4cc8f65 sorted here

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) and a new contest schedule that gives us 2 contests each year around spring and fall time, the skinning community will once again be able to unleash their inner artist with no restrictions as **there is no theme restriction for this contest.** That means that you can make a skin centred around anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
```
Comment (RockRoller01) [inline news/2024/2024-04-09-skinning-contest-unchained.md:13]: > with no restrictions as **there is no theme restriction for this contest.**

this sounds dumb, saying way too similar things right after each other

Comment (0x84f) [reply news/2024/2024-04-09-skinning-contest-unchained.md]: 423c6b51ef213b76afb6526f74345a676c68cfd0 sorted

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) and a new contest schedule that gives us 2 contests each year around spring and fall time, the skinning community will once again be able to unleash their inner artist as **there is no theme restriction for this contest.** That means that you can make a skin centred around anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
+
+Regardless of whether you are a seasoned skinning contest veteran or if this is your first time joining us, let's first get familiar with a rule change that will impact all skinning contests going forward, the proper way to submit your beautifully crafted masterpieces, and the rules of the contest!
+
+## Rule Change
```
Comment (Walavouchey) [inline news/2024/2024-04-09-skinning-contest-unchained.md:17]:
```suggestion
## Rule change
```

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+In previous contests all entries containing 3rd party assets of any kind could only be used with explicit permission from the author, which a lot of the time stifled creativity or completely stopped a skinner from realising their ideas. Starting from this contest and going forward, **any art can be used as long as the original artists are clearly credited in your submission post.**
```
Comment (Walavouchey) [inline news/2024/2024-04-09-skinning-contest-unchained.md:19]:
```suggestion
In previous skinning contests, all entries containing 3rd-party assets of any kind could only be used with explicit permission from the author, which a lot of the time stifled creativity or completely stopped a skinner from realising their ideas. Starting from this contest and going forward, **any art can be used as long as the original artists are clearly credited in your submission post**.
```

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) and a new contest schedule that gives us 2 contests each year around spring and fall time, the skinning community will once again be able to unleash their inner artist as **there is no theme restriction for this contest.** That means that you can make a skin centred around anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
+
+Regardless of whether you are a seasoned skinning contest veteran or if this is your first time joining us, let's first get familiar with a rule change that will impact all skinning contests going forward, the proper way to submit your beautifully crafted masterpieces, and the rules of the contest!
```
Comment (Walavouchey) [inline news/2024/2024-04-09-skinning-contest-unchained.md:15]: this essentially mentions the rules twice, but there's a big heading for that right below

```suggestion
Regardless of whether you are a seasoned skinning contest veteran or if this is your first time joining us, let's first get familiar with the rules as well as the proper way to submit your beautifully crafted masterpieces!
```

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+date: 2024-04-09 15:00:00 +0000
+---
+
+Showcase your unshackled creativity to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) and a new contest schedule that gives us 2 contests each year around spring and fall time, the skinning community will once again be able to unleash their inner artist as **there is no theme restriction for this contest.** That means that you can make a skin centred around anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
```
Comment (Walavouchey) [inline news/2024/2024-04-09-skinning-contest-unchained.md:13]: feels out of place and makes the sentence too long

```suggestion
With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement), the participants will once again be able to unleash their inner artists as **there is no theme restriction for this contest**. This means you can make a skin centred around anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).

Not only is the (lack of) theme reminiscent of the past, but the contest schedule will also return to a former timetable, namely giving us *two contests per year*.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-04-09-skinning-contest-unchained.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Showcase your unshackled creativity to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
```
Comment (Walavouchey) [inline news/2024/2024-04-09-skinning-contest-unchained.md:11]:
```suggestion
The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme... which is that there isn't one. Yes, you heard that right!
```

**Review by Walavouchey — DISMISSED** [review]

**Comment by RockRoller01** [comment]
Gave it a final read and its getting my seal of approval now

File: news/2024/2024-04-09-skinning-contest-unchained.md (hunk 4 above)
Comment (peppy) [reply news/2024/2024-04-09-skinning-contest-unchained.md:25]: Might be worth making that very clear if not already (due to the mention of anime girls like three times in the news post I foresee anime girls).

Comment (0x84f) [reply news/2024/2024-04-09-skinning-contest-unchained.md:25]: i added a separate heading specifying the changed rules which goes over this, is this ok?

**Review by peppy — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**